src/tasks/AotCompilerTask/MonoAOTCompiler.csproj

```diff
@@ -23,11 +23,6 @@
     <Compile Include="..\Common\IsExternalInit.cs" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
     <Compile Include="$(RepoRoot)src\libraries\System.Private.CoreLib\src\System\Diagnostics\CodeAnalysis\NullableAttributes.cs" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
   </ItemGroup>
-  <ItemGroup>
-    <Content Include="MonoAOTCompiler.props">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
What was the previous purpose of `<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>`

Reply:
It always outputed the .props file next to the AOTCompiler task assembly upon build. 
However, we don't seem to reference it from anywhere, so I removed it.